cgi-bin/sitemap.js

```diff
@@ -10,12 +10,61 @@
  * governing permissions and limitations under the License.
  */
 const algoliasearch = require('algoliasearch');
+const pick = require('lodash.pick');
+const rp = require('request-promise-native');
 const { logger } = require('@adobe/openwhisk-action-logger');
 const { wrap } = require('@adobe/openwhisk-action-utils');
 const Downloader = require('@adobe/helix-pipeline/src/utils/Downloader.js');
 const { MountConfig, IndexConfig } = require('@adobe/helix-shared');
 const { getOriginalHost } = require('../src/utils');
 
+/**
+ * Create the search provider that will return all paths for the sitemap.
+ *
+ * @param {object} index index configuration
+ * @param {object} params parameters relevant for the provider
+ * @returns provider instance
+ */
+function createSearchProvider(index, params) {
+  const { owner, repo } = params;
+
+  const { sitemap } = index;
+  if (sitemap) {
```

Comment:
is the `cgi-bin/sitemap.js` used for anything else than the sitemap? 
why does it need the _fallback_ to algolia?

Reply:
Because we want to keep our options open as long as we're not entirely sure yet which way to go...